scripts/dts/gen_defines.py

```diff
@@ -636,6 +634,13 @@ def escape(s):
     return s.replace("\\", "\\\\").replace('"', '\\"')
 
 
+def str2str(s):
+    # Put quotes around 's' and escape any double quotes and
```

Comment:
("Puts quotes around 's' and escapes any double quotes...", just to have a consistent tense with other doc comments.)